src/jvm_config.cpp

```diff
@@ -0,0 +1,104 @@
+#include <core/os/os.h>
+#include "jvm_config.h"
+
+// If changed, remember to change also TransferContext::bufferCapacity on JVM side
+const int DEFAULT_SHARED_BUFFER_SIZE{20'000'000};
+const int DEFAULT_JVM_GARBAGE_COLLECTOR_THREAD_PERIOD{500};
+
+JvmConfig::JvmConfig(const String& p_jvm_debug_port, const String& p_jvm_debug_address, const String& p_jvm_jmx_port,

```

Comment:
Is it really common to format constructor calls like this? I find this incredibly hard to read.

Reply:
It is, but we can also format it a bit better, that's now the case

Reply:
Thx